admin/data/sql/migrations/086.do.create-pupil-census-table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE [mtc_admin].[pupilCensus] (
```

Comment:
Is there a reason to NOT make this a generic table?  I thought we were going to go with `job` and not tie it in tightly with `pupilCensus`

Reply:
@jon-shipley I believe @GuyHarwood wanted to keep this as pupil census and not generic?
I will double check with him